src/05-derivatives/03-imaging.md

```diff
@@ -0,0 +1,368 @@
+```
+
+See [Anatomical Labels](#anatomical-labels) for reserved key values for `label`.
+
+### Probabilistic Segmentations
+
+Probabilistic segmentations of brain tissue represent a single tissue class with
+values ranging from 0 to 1 in individual 3D volumes or across multiple frames.
+Similarly to a discrete, binary segmentation, the label key can be used to
+specify the corresponding structure.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        func|anat|dwi/
+            <source_entities>[_space-<space>][_res-<label>][_den-<label>][_label-<label>]_probseg.nii.gz
+```
+
+Example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-orig_label-BG_probseg.nii.gz
+            sub-001_space-orig_label-WM_probseg.nii.gz
+```
+
+See [Anatomical labels](#anatomical-labels) for reserved key values for `label`.
+
+A 4D probabilistic segmentation, in which each frame corresponds to a different
+tissue class, must provide a label mapping in its JSON sidecar. For example:
+
+```Text
+pipeline/
+    sub-001/
+	    anat/
+		    sub-001_space-orig_probseg.nii.gz
+		    sub-001_space-orig_probseg.json
+```
+
+The JSON sidecar must include the label-map key that specifies a tissue label
+for each volume:
+
+```JSON
+{
+    "LabelMap": [
+        "BG",
+        "WM",
+        "GM"
+        ]
+}
+```
+
+Values of `label` need to map to Abbreviations defined in [Anatomical Labels](#anatomical-labels).
+
+### Surface Parcellations
+
+Discrete parcellations (surface segmentations) of cortical structures should be
+stored as GIFTI or CIFTI file.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        anat/
+            <source_entities>[_hemi-{L|R}][_space-<space>][_res-<label>][_den-<label>]_dseg.{label.gii|dlabel.nii}
+```
+
+The REQUIRED extension for GIFTI parcellations is `.label.gii`. The `hemi` tag is
+REQUIRED for GIFTI files. For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_hemi-L_dseg.label.gii
+            sub-001_hemi-R_dseg.label.gii
+```
+
+The REQUIRED extension for CIFTI parcellations is `.dlabel.nii`. For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_dseg.dlabel.nii
+            sub-001_dseg.dlabel.nii
+```
+
+### Anatomical Labels
+
+BIDS supplies a standard, generic label-index mapping, defined in the table
+below, that contains common tissue classes and can be used to map segmentations
+(and parcellations) between lookup tables.
+
+| Integer value | Description             | Abbreviation (label) |
+| ------------- | ----------------------- | -------------------- |
+| 0             | Background              | BG                   |
+| 1             | Gray Matter             | GM                   |
+| 2             | White Matter            | WM                   |
+| 3             | Cerebrospinal Fluid     | CSF                  |
+| 4             | Gray and White Matter   | GWM                  |
```

Comment:
So I traced this all the way back to 58dcce72a20be8f4a2367c670777dbb9579af003, don't know to what extent it has been considered critically since then, but "`GWM`" stands out to me. For a list of discrete tissue labels, it seems unusual to have a label dedicated to the combination of two specific tissues, but not labels dedicated to the combination of any other pair of tissues. If an application requires the capability to represent the combination of gray and white matter in a single voxel, then a continuous rather than discrete segmentation should be used.

Reply:
I think I agree here. Maybe somebody has a particular use case that justifies the exceptional label?

Reply:
Yeah, I think we went a little around in circles at the meeting but my view is that because volume and surface is implied by the files extension - we do not need separate dseg and dparc suffixes. I felt like the emergence of dseg and dparc was really following a freesurfer convention we didn't devote much energy to changing.  I still believe we need at least three suffixes for three different cases:

1. If the only there is only one label (i.e. data in the file is binary (0 or 1) --> "_mask.*"
2. If there can be multiple discrete integer vlabels --> "dseg.nii.gz" or "dparc.label.gii"
       --> in this case a sidecar labels.tsv file is usually required to provide interger label names
3. If the values can be floating points between 0 and 1 ---> "probseg.*"

The idea that "dseg" might be reserved for tissue differences and "dparc" might be used within a tissue does make sense though. Although I feel it might be a recommendation many people forget about and fail to use.

Don't know if that helps - or if it's just one more voice.

Reply:
Linking to https://github.com/bids-standard/bids-specification/pull/265#discussion_r432818644, because I think that last comment was meant to go there.

Reply:
Resolved in favor of dropping GWM and shifting all higher-index structures down by one.